nuxt.config.ts

```diff
@@ -130,6 +130,7 @@ export default defineNuxtConfig({
     },
   },
   runtimeConfig: {
+    googleMapsServerApiKey: process.env.GOOGLE_MAPS_API_KEY_SERVER || '',
```

Comment:
You shouldn't specify process.env here, read more https://nuxt.com/docs/4.x/guide/going-further/runtime-config